x/evm/keeper/keeper.go

```diff
@@ -310,13 +310,15 @@ type executeEVMFromCosmWasm struct {
 		CompassID string `json:"compass_id"`
 	} `json:"target_contract_info"`
 
-	// TODO: we need to have this as a payload
-	Payload string `json:"payload"`
+	Payload []byte `json:"payload"`
```

Comment:
Does this mean it will be an array like `[92,183,148,210,0,134,193,229,48,158,88,192...]`? Because there are possibly more efficient encodings.

Reply:
Nope. Golang's json de/encoder does that with base64. How is rust handling this?

Reply:
Base 64. Just wondering because the comment on this function implies it might be the array format.

Reply:
Good point. Now that you're pointing this out to me, it seems that I have put string there for a reason because when I was testing things that array thingy for the eth_address caught my attention as well. Could you please write a super simple smart contract that emits the event for executing an EVM from and simply use payload as a byte array? Basically, to use the same types that you'd use when preparing EVM call.

Reply:
https://github.com/palomachain/paloma-rs/pull/47